### PR TITLE
Remove passenger

### PR DESCRIPTION
**Pull request by thej-rayasam**

### CODE REVIEW

**Review by vishnurajasekar25 — APPROVED** [review]
Working good!

**Review by swethameenakshinatarajan — APPROVED** [review]

File: src/remove_passenger.c [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            fgets(str, 1024, fp);
+            if (!feof(fp))
+            {
+                ctr++;
+                if (ctr != delete)
+                {
+                    fprintf(fp1, "%s", str);
+                }
+            }
+        }
```
Comment (Spandukm) [inline src/remove_passenger.c]: Can you please change the positions of the curly brackets of these selected lines, place it immediate after the while condition, as we have followed for rest of the code.

Comment (thej-rayasam) [reply src/remove_passenger.c]: Updated !!

**Review by Spandukm — APPROVED** [review]
Approved

**Pull request merged**